zerver/lib/integrations.py

```diff
@@ -94,6 +96,40 @@ def add_doc_context(self, context):
         # type: (Dict[Any, Any]) -> None
         self.doc_context = context
 
+class BotIntegration(Integration):
+    DEFAULT_LOGO_STATIC_PATH_PNG = 'static/images/bots/{name}.png'
+    DEFAULT_LOGO_STATIC_PATH_SVG = 'static/images/bots/{name}.svg'
```

Comment:
The bot logos is a somewhat messy issue; not every bot will have an obvious default logo (e.g. a unit converter), and I don't want creating a logo to be a requirement for having a bot merged.  But we do need something to show in /integrations for the bot's avatar.  I think maybe we need a default logo for a built-into-Zulip bot for the /integrations page.  Maybe we could do something cute with a robot whose torso is a Zulip logo or something?

@rishig @jackrzhang @vaidap thoughts?